source/matplot/backend/backend_interface.h

```diff
@@ -213,7 +214,31 @@ namespace matplot {
             /// This is useful when tracing the gnuplot commands
             /// and when generating a gnuplot file.
             virtual void include_comment(const std::string &text);
-        };
+        }; // class backend_interface
+
+        /// Captures (backend) version information.
+        struct Version {
```

Comment:
This needs MATPLOT_EXPORTS if it's part of the API.

Reply:
It would also be good to maintain the snake_case convention. `version` or maybe `gnuplot::version` since there's no other backend.

Reply:
I'll make it snake case, sorry.
I could also move Version to gnuplot, I just thought the class is more general than just gnuplot.

Reply:
Yes. Outside `gnuplot` is also OK. I was thinking about the name-grabbing, but that's also OK. We won't need `matplot::version` for anything else.